src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/incentives/IncentivesService.kt

```diff
@@ -59,14 +59,14 @@ class IncentivesService(
       }
 
       val mapWithNomisId = telemetryMap
-        .plus(Pair("nomisBookingId", nomisResponse.nomisBookingId.toString()))
-        .plus(Pair("nomisIncentiveSequence", nomisResponse.nomisIncentiveSequence.toString()))
+        .plus(Pair("nomisBookingId", nomisResponse.bookingId.toString()))
+        .plus(Pair("nomisIncentiveSequence", nomisResponse.sequence.toString()))
```

Comment:
I'm guessing this could have been written as 
`
 telemetryMap + ("nomisIncentiveSequence" to nomisResponse.bookingId.toString() )
`
?